docs/kh2/file/type/libretto.md

```diff
@@ -0,0 +1,27 @@
+# [Kingdom Hearts II](../../index.md) - Libretto
+
+This is the file that connects the Text ID in MSG files with Talk Message in ARD files. The world-specific libretto files is a BAR file with only 1 subfile while 13libretto.bin is the subfile directly. The structure below assumes that you have extracted said subfile.
+
+## Header
+
+| Offset | Variable Type | Description |
```

Comment:
Can you rename 'Variable Type' to 'Type' as everywhere else in the documentation it's called 'Type' also please the table layout ;)

Reply:
`Variable Type` has been renamed. I don't understand what you mean by the table layout.

Reply:
![afbeelding](https://user-images.githubusercontent.com/13855042/128538152-c4dc44de-b22f-4076-a8a2-89e93cf09609.png)
Not a requirement but I prefer if the table column lines are straight.

Reply:
For some reason the preview has the table unaligned but the editor has it aligned.